### PR TITLE
This is Zac Fowler and my introduction

### PR DESCRIPTION
**Pull request by RealZacharyFowler**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/osci_2023/zacharyfowler.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Zachary Fowler
+short_name:
```
Comment (dtaivpp) [inline _authors/osci_2023/zacharyfowler.markdown:3]: Can you please put your github id into `short_name` too? We just need something in that field.

**Review by dtaivpp — CHANGES_REQUESTED** [review]

File: _authors/osci_2023/zacharyfowler.markdown (hunk 1 above)
Comment (RealZacharyFowler) [reply _authors/osci_2023/zacharyfowler.markdown:3]: Yes I will do that right now. Thanks for getting back to me.

Comment (RealZacharyFowler) [reply _authors/osci_2023/zacharyfowler.markdown]: Hello, I just pushed the requested changes. Let me know if you need anything else from me! Thanks!

**Comment by dtaivpp** [comment]
Hey @RealZacharyFowler actually could you remove the Instagram? That field can only handle twitter handles as it generates links automatically and the string there will break our build.

**Comment by krisfreedain** [comment]
never heard back from @RealZacharyFowler - closing

**Pull request closed without merging**